cpp/src/ml_mg_utils.h

```diff
@@ -42,7 +42,7 @@ void chunk_to_device(const OutType *ptr, T n, int D, int *devices,
                      cudaStream_t stream) {
   size_t chunk_size = MLCommon::ceildiv<size_t>((size_t)n, (size_t)n_chunks);
 
-#pragma omp parallel for
+  #pragma omp parallel for
```

Comment:
Why is this change part of this PR?

Reply:
I am not sure. Perhaps I made an accidental change to this file and it got included in a `make fix-format`